docs/user_guide/Upgrade_SIMP/Version_Maps/6.5.0_6.6.0.inc

```diff
@@ -0,0 +1,228 @@
+.. _upgrade-6.5.0-to-6.6.0:
+
+Upgrading from SIMP-6.5.0 to SIMP-6.6.0
+---------------------------------------
+
+.. contents:: :local:
+   :depth: 3
+
+.. _before-upgrading-to-6.6.0:
+
+Before Upgrading to 6.6.0
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SSSD Hiera Changes
+""""""""""""""""""
+
+There are two changes for systems that use :term:`SSSD`:
+
+Set simp::sssd::client::ldap_server_type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If :term:`LDAP:` and :term:`SSSD` are both used, you must set the type of LDAP
+server in :term:`hiera` using the :code:`simp::sssd::client::ldap_server_type`
+parameter.
+
+If the system is using an OpenLDAP server, you would set this option to
+:code:`plain`. For 389-DS-derived systems (389-DS, FreeIPA, RHDS, etc...) you
+would set this to :code:`389ds`. An example is provided below
+
+.. code-block:: yaml
+
+   # The upstream server is OpenLDAP
+   simp::sssd::client:ldap_server_type: 'plain'
+
+   # The upstream server is 389-DS
+   simp::sssd::client:ldap_server_type: '389ds'
+
+Requirement for SSSD >= 1.16.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In systems using :program:`sssd` prior to 1.16.0, the daemon would fail to start
+without a :code:`LOCAL` domain. For security and supportability reasons, SIMP no
+longer supports :program:`sssd` prior to 1.16.0. Likewise, the
+:code:`simp::sssd::client` class no longer configures a :code:`LOCAL` domain.
+
+If you have configured a :code:`LOCAL` domain for some other reason you must:
+
+* Remove :code:`LOCAL` from the list of domains in :term:`hiera` from
+  :code:`sssd::domains:`
+* Disable the creation of the local domain by :code:`simp::sssd::client`
+
+.. code-block:: yaml
+
+   # These are examples of the sssd::domains entry you might see in hiera.
+
+   # LOCAL is the only entry, remove everything
+   sssd::domains:
+     - LOCAL
+
+   # LOCAL is not the only entry, only remove LOCAL
+   sssd::domains:
+     - LOCAL #remove this
+     - MY_DOMAIN
+     - SOMETHING_ELSE
+
+If there are errors in systems before SIMP 6.6.0 has been fully installed, the
```

Comment:
I would just tell the user to set this setting in the bullet that says 'Disable the creation of the local domain'.   I would have the hiera block say '# The following is a workaround needed during upgrade' just like other hiera settings that are temporary.